lib/grizzly/commands/table.ex

```diff
@@ -315,7 +315,35 @@ defmodule Grizzly.Commands.Table do
       {:rssi_get, {Commands.RssiGet, handler: {WaitReport, complete_report: :rssiReport}}},
       # Zwaveplus Info
       {:zwaveplus_info_get,
-       {Commands.ZwaveplusInfoGet, handler: {WaitReport, complete_report: :zwaveplus_info_report}}}
+       {Commands.ZwaveplusInfoGet, handler: {WaitReport, complete_report: :zwaveplus_info_report}}},
+      # Schedule Entry Lock
+      {:schedule_entry_lock_week_day_get,
+       {Commands.ScheduleEntryLockWeekDayGet,
+        handler: {WaitReport, complete_report: :schedule_entry_lock_week_day_report}}},
+      {:schedule_entry_lock_year_day_get,
+       {Commands.ScheduleEntryTypeSupportedGet,
+        handler: {WaitReport, complete_report: :schedule_entry_lock_year_day_report}}},
+      {:schedule_entry_type_supported_get,
```

Comment:
Should be `{:schedule_entry_lock_type_supported_get,`

Reply:
I can't find :schedule_entry_lock_type_supported_get in the documentation

Reply:
You're right. Ignore.